package.json

```diff
@@ -36,7 +38,12 @@
     "format": "remark . -qfo && prettier . --write && xo --fix",
     "test-api": "tape \"packages/*/test.js\" \"packages/*/test/**/*.js\"",
     "test-coverage": "nyc --reporter lcov tape \"packages/*/test.js\" \"packages/*/test/**/*.js\"",
-    "test": "npm run format && npm run test-coverage"
+    "test:types": "run-p test:types:rehype-katex test:types:rehype-mathjax test:types:remark-html-katex test:types:remark-math",
+    "test:types:rehype-katex": "dtslint packages/rehype-katex/types",
+    "test:types:rehype-mathjax": "dtslint packages/rehype-mathjax",
+    "test:types:remark-html-katex": "dtslint packages/remark-html-katex/types",
+    "test:types:remark-math": "dtslint packages/remark-math/types",
+    "test": "run-s format test:types test-coverage"
```

Comment:
This seems a bit verbose to me. Would `dtslint packages/*/types` or so work too? If so, I’d prefer it instead of run-all and separate scripts.

Reply:
The problem here is that only `rehype-mathjax` has multiple type definition files that must be placed in the same directory as the original file, and that `dtslint packages/*/types` will fail because there is no `types` dir in `rehype-mathjax`.
Possible solutions are:
1. Move `test-types` to each package and run `lerna run test-types` in the root.